firebase-firestore/src/test/java/com/google/firebase/firestore/local/CountingQueryEngine.java

```diff
@@ -84,9 +85,9 @@ QueryEngine getSubject() {
 
   /**
    * Returns the number of documents returned by the RemoteDocumentCache's
-   * `getDocumentsMatchingQuery()` API (since the last call to `resetCounts()`)
+   * `getAll()` API (since the last call to `resetCounts()`)
    */
-  int getDocumentsReadByQuery() {
+  int getDocumentsReadByColllection() {
     return documentsReadByQuery[0];
```

Comment:
Should the `documentsReadByQuery` instance variable be renamed to `documentsReadByCollection`, with only 2 L's? :P

Reply:
Yes, thanks for catching.